content/source-git/design.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Source-git design
+date: 2018-12-31T23:59:59+01:00
+weight: 1
+---
+
+# Source-git design
+
+This document serves as a detailed description of source-git. Please bear in
+mind that some things are a subject to change. Paragraphs marked with "‼️💣️" are
+known to require further work to be better defined.
+
+**Authors**: Stef Walter, Tomas Tomecek  
+**Updates by:** Hunor Csomortáni
```

Comment:
I'd make all three of us authors (or drop it since it's in git now - I recall we had this header because it was a gdoc)